src/language/concurrency.md

```diff
@@ -237,303 +287,272 @@ After handling the events, the isolate exits.
+[`Finalizer`]: {{site.dart-api}}/{{site.data.pkg-vers.SDK.channel}}/dart-core/Finalizer-class.html
+[`NativeFinalizer`]: {{site.dart-api}}/{{site.data.pkg-vers.SDK.channel}}/dart-ffi/NativeFinalizer-class.html
+[`Pointer`]: {{site.dart-api}}/{{site.data.pkg-vers.SDK.channel}}/dart-ffi/Pointer-class.html
+[`UserTag`]: {{site.dart-api}}/{{site.data.pkg-vers.SDK.channel}}/dart-developer/UserTag-class.html
 
 <a id="web"></a>
 ## Concurrency on the web
 
 All Dart apps can use `async-await`, `Future`, and `Stream`
-for non-blocking, interleaved computations.
-The [Dart web platform][], however, does not support isolates.
-Dart web apps can use [web workers][] to
-run scripts in background threads
-similar to isolates.
-Web workers' functionality and capabilities
-differ somewhat from isolates, though.
-
-For instance, when web workers send data between threads,
-they copy the data back and forth.
-Data copying can be very slow, though,
-especially for large messages. 
-Isolates do the same, but also provide APIs
-that can more efficiently _transfer_
+for non-blocking, interleaved computations. The [Dart web platform][], however,
+does not support isolates. Dart web apps can use [web workers][] to run scripts
+in background threads similar to isolates. Web workers' functionality and
+capabilities differ somewhat from isolates, though.
+
+For instance, when web workers send data between threads, they copy the data
+back and forth. Data copying can be very slow, though, especially for large
+messages. Isolates do the same, but also provide APIs that can more efficiently
+_transfer_
 the memory that holds the message instead.
 
-Creating web workers and isolates also differs.
-You can only create web workers by declaring
-a separate program entrypoint and compiling it separately.
-Starting a web worker is similar to using `Isolate.spawnUri` to start an isolate.
-You can also start an isolate with `Isolate.spawn`,
-which requires fewer resources because it
+Creating web workers and isolates also differs. You can only create web workers
+by declaring a separate program entrypoint and compiling it separately. Starting
+a web worker is similar to using `Isolate.spawnUri` to start an isolate. You can
+also start an isolate with `Isolate.spawn`, which requires fewer resources
+because it
 [reuses some of the same code and data](#performance-and-isolate-groups)
-as the spawning isolate. 
-Web workers don't have an equivalent API.
+as the spawning isolate. Web workers don't have an equivalent API.
 
 [Dart web platform]: /overview#platform
 [web workers]: https://developer.mozilla.org/docs/Web/API/Web_Workers_API/Using_web_workers
+
+
+## Additional resources
+
+- If you’re using many isolates, consider
+  the [IsolateNameServer][]
+  in Flutter, or
+  the [pub package][] that clones
+  the functionality for Dart applications not using Flutter.
+- Dart’s isolates are an implementation of
```

Comment:
This reads less like an "Additional resource" and more like an instructional sentence. Try to keep the entries in a list parallel in their construction.